scripts/commands/delinventory.lua

```diff
@@ -0,0 +1,38 @@
+-----------------------------------
+-- func: delcontaineritems
```

Comment:
PR title, filename, descriptor etc. are all out of line -> `delContainerItems`

Reply:
Updated to reflect it now being `delinventory`.